### ISSUE TITLE
Error: [$sce:unsafe] when in production

### ISSUE THREAD

**Issue by edencorbin** [fraywing/textAngular#1127]
textAngular is working great for me on localhost, however when I run in production I am getting an error of Error: [$sce:unsafe], I'm thinking this is related to textAngular-sanitize.min.js. Any ideas how I might fix this?

**Comment by pbassut** [comment]
You're probably loading something from another domain? If so, that's not allowed by angular unless you wrap the url in ```$sce.trustAsUrl```

**Comment by edencorbin** [comment]
Okay, your right it's not directly textAngular. Interestingly it's not loading something from another domain I don't think. I'm using sails stack, and it appears if I run the app in production, when invokes minification/etc. through grunt, I get the error. I'll pick up the thread on a sails.js forum, or perhaps look at loading the textAngular in via cdn or otherwise excluding from the minification process.

**Comment by pbassut** [comment]
I can help you if you paste the error here.

**Comment by edencorbin** [comment]
Well, finally, I resolved it, actually by taking a different simpler approach, cutsomizing toolbar is working fine via html/view as follows (for example):
ta-toolbar="[['h1','h2','h3'],['bold','italics']]"

Problem solved. However, out of interest, I'm nearly certain my issue has to do with this: http://thegreenpizza.github.io/2013/05/25/building-minification-safe-angular.js-applications/

Here was the code that worked in production but not minified:

`var myApp =  angular.module('myApp', ['datatables', 'textAngular']).
config(function($provide) {
  $provide.decorator('taOptions', ['$delegate', function(taOptions) { 
    taOptions.toolbar = [
        ['pre', 'bold', 'italics', 'underline', 'strikeThrough','ol','insertLink', 'insertImage','html'],
    ];
    return taOptions;
  }]);
});`

Much as I tried I could not properly formatt this to work and be minimizable, if its immedietely obviouse to you that would be highly interesting, but again problem solved, thanks for the awesome plugin, looking forward to using it in a production application soon.
